irods/manager/user_manager.py

```diff
@@ -16,6 +16,22 @@
 
 class UserManager(Manager):
 
+    def remove_quota(self, user_name, resource = 'total'):
+        amount = 0
+        message_body = GeneralAdminRequest(
+            "set-quota",
+            "user",
+            user_name,
+            resource,
+            str(amount)
+        )
+        request = iRODSMessage("RODS_API_REQ", msg=message_body,
+                               int_info=api_number['GENERAL_ADMIN_AN'])
+        with self.sess.pool.get_connection() as conn:
+            conn.send(request)
+            response = conn.recv()
+        logger.debug(response.int_info)
+
```

Comment:
This can be removed now, correct?

Reply:
Perhaps we should keep it so that folks can remove user quotas previously set, ie. do the equivalent of `iadmin suq username total 0`?

Reply:
True. Proceed.